pkg/generate/app/dockerimagelookup.go

```diff
@@ -76,7 +76,7 @@ func (r DockerClientSearcher) Search(terms ...string) (ComponentMatches, error)
 				termMatches = append(termMatches, matches...)
 			case ErrNoMatch:
 			default:
-				return nil, err
+				glog.V(1).Infof("An error occurred searching the Docker registry: %v", err)
```

Comment:
> Is there a possibility I run new-app and get no output now because of this? If so, we may want to output by default.

You will always get an output, saying that we couldn't find a match for your input.